draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1827,6 +1827,14 @@ that are mandatory to implement for CT-using TLS clients (see
 if it did not include both the `transparency_info` and `status_request` TLS
 extensions in the ClientHello.
 
+## TLS Server {#no_tls_servers}
```

Comment:
I think this new section should be a subsection of the existing "TLS Server" section (search for "# TLS Servers {#tls_servers}" in the .md source).  Perhaps name this new section "TLS Client Authentication"?